.github/workflows/tests.yaml

```diff
@@ -14,11 +14,6 @@ on:
         type: string
         default: 'main'
 
-
```

Comment:
Seems like these start to interfere because they are nesting from the general CI flow. 
Should be fine to keep them running to completion since either the parent group will cancel them on newer runs or you manually triggered it anyways.